pkg/eraser/eraser.go

```diff
@@ -169,8 +170,33 @@ func updateStatus(ctx context.Context, clientset *kubernetes.Clientset, results
 		Name(imageStatus.Name).
 		Resource("imagestatuses").
 		Body(body).DoRaw(ctx)
-
```

Comment:
Maybe `if !apierrors.IsAlreadyExists(err) { return err }` here and de-indent the block below?